File: app/src/main/java/com/facundoaramayo/testrappikotlin/data/AppConfig.java
package com.facundoaramayo.testrappikotlin.data;

public class AppConfig {

    // flag for save image offline
    public static final boolean IMAGE_CACHE = true;

    // if place data more than 200 items set TRUE
    public static final boolean LAZY_LOAD = false;

    // when user enable gps, places will sort by distance
    public static final boolean SORT_BY_DISTANCE = true;

    // distance metric, fill with KILOMETER or MILE only
    public static final String DISTANCE_METRIC_CODE = "KILOMETER";

    // related to UI display string
    public static final String DISTANCE_METRIC_STR = "Km";


}

File: app/src/main/java/com/facundoaramayo/testrappikotlin/data/AppConfig.kt
package com.facundoaramayo.testrappikotlin.data

object AppConfig {

    // flag for save image offline
    const val IMAGE_CACHE = true

    // if place data more than 200 items set TRUE
    const val LAZY_LOAD = false

    // when user enable gps, places will sort by distance
    const val SORT_BY_DISTANCE = true

    // distance metric, fill with KILOMETER or MILE only
    const val DISTANCE_METRIC_CODE = "KILOMETER"

    // related to UI display string
    const val DISTANCE_METRIC_STR = "Km"


}
